src/components/Booking/BookingContextProvider.jsx

```diff
@@ -10,26 +10,35 @@ export function useBookingContext() {
 }
 
 function BookingContextProvider({ children }) {
-  const [selectedDate, setSelectedDate] = useState(null);
+  const [selectedDate, setSelectedDate] = useState(findTodayDate());
   const [weekDaysDetails, setWeekDaysDetails] = useState(null);
-
   const [daySlotMapping, setDaySlotMapping] = useState([]);
+  const [selectedTimeslot,setselectedTimeslot] = useState([]);
+ 
 
   useEffect(() => {
     if (weekDaysDetails && selectedDate) {
+     
       const mappedSlots = weekDaysDetails
         .filter(element => element.days_of_week.includes(format(selectedDate, 'EEEE').toUpperCase().substring(0, 3)))
```

Comment:
format(selectedDate, 'EEEE').toUpperCase().substring(0, 3)))
Move the logic to date help function

Reply:
Date helper missing